components/commons/modals/FolderDeleteModal.tsx

```diff
@@ -0,0 +1,19 @@
+import CloseButton from "./CloseButton";
+import { DeleteModalModalBox, ModalLayout, RedButton } from "./styledModal";
+
+function FolderDeleteModal({ closeModal }: { closeModal: () => void }) {
```

Comment:
이런 경우는 CloseButton의 코멘트와 달리, onClick보다는 좀 더 명확해도 자연스러울거같습니다.
왜냐하면 FolderDeleteModal 이라는 이름이 명확한 도메인/기능을 담고있으니까요.
```suggestion
function FolderDeleteConfirmModal({ onConfirm }: { onConfirm: () => void }) {
// 폴더 삭제를 물어보는 모달이고, '확인'하면 onConfirm을 호출해줄거라는 의도를 드러냄
```
```suggestion
function FolderDeleteModal({ id, onDeleteFolder }: { id, onDeleteFolder: () => void }) {
// 폴더 삭제모달이 이름에 걸맞게 알아서 해당 폴더 `id`를 찾아서 삭제해줄거고, 그 후에 뭔가(onDeleteFolder) 해줄 거라는 의도를 드러내는 props
```